SpywareFilter/sections/general_extensions.txt

```diff
@@ -31,6 +31,9 @@ apteka.ru#%#window.yaCounter14913877={ reachGoal: function() {} };
 popmech.ru#%#AG_onLoad(function() { window.POPMECH.increaseCounters = function() {} });
 ! https://forum.adguard.com/index.php?threads/resolved-accountkiller-com.16162/
 accountkiller.com#%#window.ga = function(){var a = arguments[5];a&&a.hitCallback&&a.hitCallback();}
+! https://github.com/AdguardTeam/AdguardFilters/issues/54102
+!+ NOT_PLATFORM(ios, ext_android_cb, ext_safari, ext_ublock)

```

Comment:
No need to add this directive in case of `$redirect` and content blockers(ios, ext_android_cb, ext_safari)

Reply:
> 
> 
> Btw why it is not reproduced in uBO?

uBlock Privacy filter takes redirect-by-default approach for GTM.

Reply:
> uBlock Privacy filter takes redirect-by-default approach for GTM.

In this case `@@||googletagmanager.com/gtm.js$domain=theverge.com` should not be removed(will work in content blockers, and redirect rule in full AdGuards)

Reply:
Tested on Yandex browser + CB, the whitelist didn't change the issue. Do you mean anyway the whitelist is needed for other reason?